Module/STIG/Convert/Functions.PowerStigXml.ps1

```diff
@@ -698,4 +707,148 @@ function Get-StigObjectsWithOrgSettings
     $ConvertedStigObjects |
         Where-Object { $PSitem.OrganizationValueRequired -eq $true}
 }
+
+function Get-OrgSettingPropertyFromStigRule
+{
+    [CmdletBinding()]
+    [OutputType([string[]])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [psobject]
+        $ConvertedStig
+    )
+
+    $propertiesToRemove = Get-BaseRulePropertyNames
+    [System.Collections.ArrayList] $rulePropertyNames = (Get-Member -InputObject $ConvertedStig -MemberType Property).Name
+    foreach ($property in $propertiesToRemove)
+    {
+        $rulePropertyNames.RemoveAt($rulePropertyNames.IndexOf($property))
+    }
+    foreach ($propertyName in $rulePropertyNames)
+    {
+        if ([string]::IsNullOrEmpty($ConvertedStig.$propertyName))
+        {
+            [array] $orgSettingProperties += $propertyName
+        }
+    }
+
+    return $orgSettingProperties
+}
+
+function Get-HardCodedRuleLogFileEntry
```

Comment:
This needs legitimate comment based help with examples since this is a public function.

Reply:
Done